proto/ignition/msgs/scene.proto

```diff
@@ -51,4 +57,7 @@ message Scene
 
   /// \brief Show/hide world origin indicator.
   bool origin_visual  = 12;
+
+  /// \brief Name of shadow caster material.
```

Comment:
```suggestion
  /// \brief Shadow caster material script.
```

Reply:
applied in https://github.com/scpeters/ign-msgs/commit/e418f2f2d3b62e45b9bcc4f2cd5c5825f8108ab4